components/service/glean/src/main/java/mozilla/components/service/glean/utils/LanguageUtils.kt

```diff
@@ -0,0 +1,49 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package mozilla.components.service.glean.utils
+
+import java.util.Locale
+
+/**
+ * Gets a gecko-compatible locale string (e.g. "es-ES" instead of Java [Locale]
+ * "es_ES") for the default locale.
+ *
+ * This method approximates the API21 method [Locale.toLanguageTag].
+ *
+ * @return a locale string that supports custom injected languages.
+ */
+internal fun getLanguageTag(): String {
```

Comment:
Minor point and it might be just me.  Maybe `language` -> `locale`?  I see "language" and I think "programming language" before I think of "human language".  Plus, I think technically this function returns a locale and not a language since it includes a region.

Reply:
I renamed the Language functions to be more appropriate:
`getLocaleTag()` and `getLanguageFromLocale()`